spec/main-process/atom-application.test.js

```diff
@@ -368,6 +370,23 @@ describe('AtomApplication', function () {
     })
   })
 
+  describe('before quitting', function () {
+    it('waits until all the windows have saved their state and then quits', async function () {
```

Comment:
Yo @as-cii, is this a new feature in Atom's spec runner? I previously used `it` and stuff from [`jasmine-fix`](https://www.npmjs.com/package/jasmine-fix) to make it accept async callbacks